data/jobs.txt

```diff
@@ -1477,6 +1477,74 @@ mission "Core Pirate Attack"
 	on complete
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
+		
+mission "Raider Attack 1"
+	name "Defend <planet>"
+	description "Defeat an alien raid on <destination>."
+	minor
+	repeat
+	to offer
+		"combat rating" > 550
+		random < 7
+	source
+		planet Sheratan
+		planet Shangri-la
+		planet Nimbus
+	npc evade
+		personality timid unconstrained uninterested
+		government Korath
```

Comment:
"Korath" per style above

Personally I try to order the NPC specification
1. government
2. personality
3. system
4. fleet

but that is purely a personal style choice of mine.